content/basic-concepts.md

```diff
@@ -28,7 +28,7 @@ https:/example.com/OData.svc/[CONTENTPATH]
 
 # Query options
 
-Query options are query string parameters a client may specify to control the amount and order of the data that a service returns for the resource identified by the URI. In sensenet there're two types of query options available [OData System Query Options](https://www.odata.org/documentation/odata-version-2-0/uri-conventions/) and custom sensenet query options. The OData standard query options' names are prefixed with a "$" character, sensenet query options should be used without a prefix.
+Query options are query string parameters a client may specify to control the amount and order of the data that a service returns for the resource identified by the URI. In sensenet there're two types of query options available [OData System Query Options](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_URLComponents) and custom sensenet query options. The OData standard query options' names are prefixed with a "$" character, sensenet query options should be used without a prefix.
```

Comment:
I've changed the url to link to the latest implementation. Which version of OData are we supporting right now?

Reply:
We use OData protocol v3.